### PR TITLE
exiv2 0.25

### PR DESCRIPTION
**Pull request by dunn**
The HEAD install was just adapted from `bootstrap.linux` in the SVN repo, so let me know if there are any other tweaks I should make.

### CODE REVIEW

File: Library/Formula/exiv2.rb [hunk 1]
```diff
@@ -1,23 +1,61 @@
...
   end
 
   option :universal
 
   def install
     ENV.universal_binary if build.universal?
-    system "./configure", "--prefix=#{prefix}", "--disable-dependency-tracking"
-    system "make install"
+    if build.head?
+      args = %W[-DCMAKE_BUILD_TYPE=Release
```
Comment (DomT4) [inline Library/Formula/exiv2.rb]: We pass a lot of these automatically with `std_cmake_args`. Let's use that and then add extra `args` as necessary.

File: Library/Formula/exiv2.rb [hunk 2]
```diff
@@ -1,23 +1,60 @@
...
 
   option :universal
 
   def install
     ENV.universal_binary if build.universal?
-    system "./configure", "--prefix=#{prefix}", "--disable-dependency-tracking"
-    system "make install"
+    if build.head?
+      args = std_cmake_args
+      args += %W[-DEXIV2_ENABLE_SHARED=ON
```
Comment (DomT4) [inline Library/Formula/exiv2.rb]: It's a nit, but can we shove this onto the next line so the indentation doesn't look so out of place?

```ruby
args += %W[
  -DEXIV2_ENABLE_SHARED=ON
  -DEXIV2_ENABLE_XMP=ON
  -DEXIV2_ENABLE_LIBXMP=ON
]
```

And such.

Comment (dunn) [inline Library/Formula/exiv2.rb]: Sure!

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/114ca6f98a2c6488b3b7128553d0ffb271a42ebc. Thanks again @dunn!

**Pull request closed without merging**